src/BuildScriptGeneratorCli/Commands/BuildScriptCommand.cs

```diff
@@ -48,6 +48,12 @@ internal class BuildScriptCommand : CommandBase
             Description = "Additional information used by this tool to generate and run build scripts.")]
         public string[] Properties { get; set; }
 
+        [Option(
+            "--output",
+            CommandOptionType.SingleValue,
+            Description = "The path that the build script will be written to. If not specified, the result will be written to STDOUT.")]
```

Comment:
nit: We try to keep the width of the text <=120 characters to avoid scrolling in GitHub. I use the VS extension `Guidelines` with which you can set the limit to 120.